docs/source/runner.md

```diff
@@ -30,10 +30,12 @@ for example, `--model histopathology.PandaImageNetMIL` is effectively telling th
 
 ## Running ML experiments in Azure ML
 
-To train in AzureML, add a `--azureml` flag. Use the flag `--cluster` to specify the name of the cluster
+To train in AzureML, use the flag `--cluster` to specify the name of the cluster
 in your Workspace that you want to submit the job to. So the whole command would look like:
-`himl-runner --model=HelloWorld --cluster=my_cluster_name --azureml`. You can also specify `--num_nodes` if
-you wish to distribute the model training.
+```
+himl-runner --model=HelloContainer --cluster=my_cluster_name
```

Comment:
Think this is out of sync with main - Anton merged a PR yesterday to replace 'HelloContainer' with 'HelloWorld'